src/components/videos/Videos.test.tsx

```diff
@@ -28,23 +30,38 @@ describe("Videos page: Render", () => {
     useState: jest.fn(),
   }));
 
+  jest.spyOn(React, "useEffect").mockImplementation((f) => f());
+
   const VideoProps: Video = {
-    url: "video url",
-    user: {
-      name: "user name",
-      position: "user position",
-    },
-    dateUploaded: new Date(2022, 7, 1, 9).toLocaleString(),
+    firebaseName: "firebase url",
+    title: "video title",
     description: "video description",
   };
 
   it("should render the video component", () => {
     const snapshot = render(
       <VideoCard
         Actions={{ setHidden: jest.fn(), setModalClickedVideoData: jest.fn() }}
-        Video={VideoProps}
+        video={VideoProps}
+        disableOnClick={false}
       />,
     );
     expect(snapshot).toMatchSnapshot();
   });
+  it("should unhide video player when video clicked", () => {
+    const setHiddenMock = jest.fn();
+    render(
+      <VideoCard
+        Actions={{ setHidden: setHiddenMock, setModalClickedVideoData: jest.fn() }}
+        video={VideoProps}
+        disableOnClick={false}
+      />,
+    );
+
+    const video = screen.getByTestId(VideoProps.title);
+    expect(video).toBeInTheDocument();
+    fireEvent.click(video);
```

Comment:
Some of the docs suggest using the user-event package instead of fireEvent because it is "more real".

 This isn't a super important point but may interest you.

https://testing-library.com/docs/user-event/intro#differences-from-fireevent

https://ph-fritsche.github.io/blog/post/why-userevent

Reply:
Thanks for that Tom, that is interesting! I've changed it to use user-event here instead of fireEvent